libcxx/docs/Status/Cxx17.rst

```diff
@@ -42,7 +42,6 @@ Paper Status
 
    .. [#note-P0067] P0067: ``std::(to|from)_chars`` for integrals has been available since version 7.0. ``std::to_chars`` for ``float`` and ``double`` since version 14.0 ``std::to_chars`` for ``long double`` uses the implementation for ``double``.
    .. [#note-P0607] P0607: The parts of P0607 that are not done are the ``<regex>`` bits.
-   .. [#note-P0154] P0154: This is currently not implemented for clang because clang does not implement the required macros.
```

Comment:
When LLVM-19 is released libc++ still supports Clang 18, which does not have the needed support. Therefore I prefer to keep the note and mention it requires Clang 19 or newer.

(I've no objection to mark the feature as complete.)

Reply:
Done